StyleCop.Analyzers/Directory.Build.targets

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
 
+  <PropertyGroup>
+    <SkipAddAllResxFilesAsAdditionalFiles>true</SkipAddAllResxFilesAsAdditionalFiles>
+  </PropertyGroup>
+
   <ItemGroup>
-    <!-- Enable code generation for resource files -->
+    <!-- Enable code generation for resource files. -->
     <CodeGenerationEmbeddedResource Include="@(EmbeddedResource)" Exclude="**\*.??-??.resx" />
-    <EmbeddedResource Update="@(CodeGenerationEmbeddedResource)" Generator="ResXFileCodeGenerator" />
+    <AdditionalFiles Include="@(CodeGenerationEmbeddedResource)" GenerateSource="true" />
+
+    <NonCodeGenerationEmbeddedResource Include="@(EmbeddedResource)" Exclude="@(CodeGenerationEmbeddedResource)" />
```

Comment:
💭 Seems like this should be included in the default behavior of ResxSourceGenerator

Reply:
➡️ This is now the default behavior